rts/Game/UI/GuiHandler.cpp

```diff
@@ -1448,6 +1450,17 @@ bool CGuiHandler::SetActiveCommand(int cmdIndex, int button,
 	return retval;
 }
 
+void CGuiHandler::SetActiveCommandIndex(int newIndex)
+{
+	if (inCommand != newIndex) {
+		inCommand = newIndex;
+		if (inCommand < commands.size())
+			eventHandler.ActiveCommandChanged(&commands[inCommand]);
```

Comment:
`inCommand` is signed and can be `-1`, perhaps cast explicitly?

Reply:
added a check to make sure it's greater than 0, let me know if you still think it needs explicit casting but I think with that it shouldn't.